src/P5Functions/touchmouse.js

```diff
@@ -315,6 +315,8 @@ module.exports = function withTouch(p, curElement, attachEventHandler, document,
 
     if (delta && typeof p.mouseScrolled === 'function') {
       p.mouseScrolled();
+      e.preventDefault();
+      e.stopPropagation();
```

Comment:
note that `return false` was an older method to prevent propagation, and isn't supported by newer browsers anymore, so we definitely need these two calls. It also feels like this is the right spot: only if the user defined code for their own `mouseScrolled` handler should we kill off scroll when the mouse cursor is over the sketch canvas.